release-notes.hbs.md

```diff
@@ -367,6 +367,12 @@ This release has the following security fixes, listed by component and area.
 
 The following issues, listed by component and area, are resolved in this release.
 
+#### <a id='1-6-0-scst-store-ri'></a> Supply Chain Security Tools - Store
```

Comment:
I'm not sure where you got this list from, but out of curiosity, I did [this search in JIRA](https://jira.eng.vmware.com/browse/META-1113?jql=project%20%3D%20META%20AND%20issuetype%20%3D%20Bug%20AND%20fixVersion%20%3D%20%22TAP%201.6.0%22) and it came out with a few more bugs that were fixed for 1.6.0. Did we want to include the rest of them? Or were they intentionally omitted?

Reply:
I specifically ignored all SBOM per Builds bugs as the feature has not yet been released to the general public. So, I feel we don't need to mention those as those fixes are being shipped as part of the initial release of the feature.

Makes sense?